addons/rating/models/rating.py

```diff
@@ -56,8 +56,8 @@ def _selection_target_model(self):
     rating_image = fields.Binary('Image', compute='_compute_rating_image')
     rating_text = fields.Selection([
         ('satisfied', 'Satisfied'),
-        ('not_satisfied', 'Not satisfied'),
-        ('highly_dissatisfied', 'Highly dissatisfied'),
+        ('okay', 'Okay'),
```

Comment:
While we are changing rating terms, I would maybe reduce key length to save some extra DB space, like
  * 'top' -> Satisfied
  * 'ok' -> Okay
  * 'ko' -> Dissatistifed
  * 'none' -> No Rating Yet

On heavily used DBs this could lead to some memory gain.

If you feel it is a good idea, could be done in a separate commit as this is an additional feature of the PR.